terraform-status

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env ruby
+
+require 'net/smtp'
+require 'open3'
+
+EMAIL_TO = 'rsmith11@stanford.edu'.freeze
+
+# Given a message, send it out as an email.
+def mail_message(message)
+    Net::SMTP.start('localhost') do |smtp|
+      smtp.send_message message, 'nobody@stanford.edu', EMAIL_TO
+    end
+end
+
+tfchanges = []
+tffailed = []
+
+Dir.glob("/home/terraform/terraform/hosts/*.tf") {|tffile|
```

Comment:
Right before this, run a chdir and then terraform init? Or maybe you don't need the chdir.. the later terraform bits suggest that you're already running from inside the right directory?

https://ruby-doc.org/core-2.6.3/Dir.html

Reply:
Thank you for your help Jon.  I made the changes and now terraform init is being called within script now.